src/runtimes/node/utils/entry_file.ts

```diff
@@ -12,28 +14,72 @@ const getEntryFileContents = (mainPath: string, moduleFormat: string) => {
   const importPath = `.${mainPath.startsWith('/') ? mainPath : `/${mainPath}`}`
 
   if (moduleFormat === ModuleFormat.COMMONJS) {
-    return `module.exports = require('${importPath}')`
+    return `module.exports=require('${importPath}')`
   }
 
-  return `export { handler } from '${importPath}'`
+  return `export{handler}from'${importPath}'`
 }
 
+// They are also in the order that lambda will try to find the entry point
+const POSSIBLE_LAMBDA_ENTRY_EXTENSIONS = ['.js', '.mjs', '.cjs']
+
+export const isEntryFile = (
```

Comment:
It's not obvious to me why we need a function that checks whether a file is the entry file, since theoretically we're in control of specifying what the entry file is? Could you add a comment to make the intent more explicit?

Reply:
Added a comment.